File: app/src/main/java/com/example/testlunu/net/acu/weather/WeatherWrapper.java
package com.example.testlunu.net.acu.weather;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class WeatherWrapper {

    @SerializedName("Headline")
    @Expose
    public Headline headline;

    public Headline getHeadline() {
        return headline;
    }

    public List<DailyForecast> getDailyForecasts() {
        return dailyForecasts;
    }

    @SerializedName("DailyForecasts")
    @Expose
    public List<DailyForecast> dailyForecasts = null;

}

class AirAndPollen {

    @SerializedName("Name")
    @Expose
    public String name;
    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Category")
    @Expose
    public String category;
    @SerializedName("CategoryValue")
    @Expose
    public Integer categoryValue;
    @SerializedName("Type")
    @Expose
    public String type;

}

class Cooling {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}


class Day {

    @SerializedName("Icon")
    @Expose
    public Integer icon;
    @SerializedName("IconPhrase")
    @Expose
    public String iconPhrase;
    @SerializedName("HasPrecipitation")
    @Expose
    public Boolean hasPrecipitation;
    @SerializedName("ShortPhrase")
    @Expose
    public String shortPhrase;
    @SerializedName("LongPhrase")
    @Expose
    public String longPhrase;
    @SerializedName("PrecipitationProbability")
    @Expose
    public Integer precipitationProbability;
    @SerializedName("ThunderstormProbability")
    @Expose
    public Integer thunderstormProbability;
    @SerializedName("RainProbability")
    @Expose
    public Integer rainProbability;
    @SerializedName("SnowProbability")
    @Expose
    public Integer snowProbability;
    @SerializedName("IceProbability")
    @Expose
    public Integer iceProbability;
    @SerializedName("Wind")
    @Expose
    public Wind wind;
    @SerializedName("WindGust")
    @Expose
    public WindGust windGust;
    @SerializedName("TotalLiquid")
    @Expose
    public TotalLiquid totalLiquid;
    @SerializedName("Rain")
    @Expose
    public Rain rain;
    @SerializedName("Snow")
    @Expose
    public Snow snow;
    @SerializedName("Ice")
    @Expose
    public Ice ice;
    @SerializedName("HoursOfPrecipitation")
    @Expose
    public Integer hoursOfPrecipitation;
    @SerializedName("HoursOfRain")
    @Expose
    public Integer hoursOfRain;
    @SerializedName("HoursOfSnow")
    @Expose
    public Integer hoursOfSnow;
    @SerializedName("HoursOfIce")
    @Expose
    public Integer hoursOfIce;
    @SerializedName("CloudCover")
    @Expose
    public Integer cloudCover;

}

class DegreeDaySummary {

    @SerializedName("Heating")
    @Expose
    public Heating heating;
    @SerializedName("Cooling")
    @Expose
    public Cooling cooling;

}

class Direction {

    @SerializedName("Degrees")
    @Expose
    public Integer degrees;
    @SerializedName("Localized")
    @Expose
    public String localized;
    @SerializedName("English")
    @Expose
    public String english;

}

class Direction_ {

    @SerializedName("Degrees")
    @Expose
    public Integer degrees;
    @SerializedName("Localized")
    @Expose
    public String localized;
    @SerializedName("English")
    @Expose
    public String english;

}

class Direction__ {

    @SerializedName("Degrees")
    @Expose
    public Integer degrees;
    @SerializedName("Localized")
    @Expose
    public String localized;
    @SerializedName("English")
    @Expose
    public String english;

}

class Direction___ {

    @SerializedName("Degrees")
    @Expose
    public Integer degrees;
    @SerializedName("Localized")
    @Expose
    public String localized;
    @SerializedName("English")
    @Expose
    public String english;

}



class Headline {

    @SerializedName("EffectiveDate")
    @Expose
    public String effectiveDate;
    @SerializedName("EffectiveEpochDate")
    @Expose
    public Integer effectiveEpochDate;
    @SerializedName("Severity")
    @Expose
    public Integer severity;
    @SerializedName("Text")
    @Expose
    public String text;
    @SerializedName("Category")
    @Expose
    public String category;
    @SerializedName("EndDate")
    @Expose
    public String endDate;
    @SerializedName("EndEpochDate")
    @Expose
    public Integer endEpochDate;
    @SerializedName("MobileLink")
    @Expose
    public String mobileLink;
    @SerializedName("Link")
    @Expose
    public String link;

}

class Heating {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Ice {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Ice_ {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}



class Maximum_ {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Maximum__ {

    @SerializedName("Value")
    @Expose
    public Float value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}



class Minimum_ {

    @SerializedName("Value")
    @Expose
    public Float value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Minimum__ {

    @SerializedName("Value")
    @Expose
    public Float value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Moon {

    @SerializedName("Rise")
    @Expose
    public String rise;
    @SerializedName("EpochRise")
    @Expose
    public Integer epochRise;
    @SerializedName("Set")
    @Expose
    public String set;
    @SerializedName("EpochSet")
    @Expose
    public Integer epochSet;
    @SerializedName("Phase")
    @Expose
    public String phase;
    @SerializedName("Age")
    @Expose
    public Integer age;

}

class Night {

    @SerializedName("Icon")
    @Expose
    public Integer icon;
    @SerializedName("IconPhrase")
    @Expose
    public String iconPhrase;
    @SerializedName("HasPrecipitation")
    @Expose
    public Boolean hasPrecipitation;
    @SerializedName("ShortPhrase")
    @Expose
    public String shortPhrase;
    @SerializedName("LongPhrase")
    @Expose
    public String longPhrase;
    @SerializedName("PrecipitationProbability")
    @Expose
    public Integer precipitationProbability;
    @SerializedName("ThunderstormProbability")
    @Expose
    public Integer thunderstormProbability;
    @SerializedName("RainProbability")
    @Expose
    public Integer rainProbability;
    @SerializedName("SnowProbability")
    @Expose
    public Integer snowProbability;
    @SerializedName("IceProbability")
    @Expose
    public Integer iceProbability;
    @SerializedName("Wind")
    @Expose
    public Wind_ wind;
    @SerializedName("WindGust")
    @Expose
    public WindGust_ windGust;
    @SerializedName("TotalLiquid")
    @Expose
    public TotalLiquid_ totalLiquid;
    @SerializedName("Rain")
    @Expose
    public Rain_ rain;
    @SerializedName("Snow")
    @Expose
    public Snow_ snow;
    @SerializedName("Ice")
    @Expose
    public Ice_ ice;
    @SerializedName("HoursOfPrecipitation")
    @Expose
    public Integer hoursOfPrecipitation;
    @SerializedName("HoursOfRain")
    @Expose
    public Integer hoursOfRain;
    @SerializedName("HoursOfSnow")
    @Expose
    public Integer hoursOfSnow;
    @SerializedName("HoursOfIce")
    @Expose
    public Integer hoursOfIce;
    @SerializedName("CloudCover")
    @Expose
    public Integer cloudCover;

}

class Rain {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Rain_ {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class RealFeelTemperature {

    @SerializedName("Minimum")
    @Expose
    public Minimum_ minimum;
    @SerializedName("Maximum")
    @Expose
    public Maximum_ maximum;

}

class RealFeelTemperatureShade {

    @SerializedName("Minimum")
    @Expose
    public Minimum__ minimum;
    @SerializedName("Maximum")
    @Expose
    public Maximum__ maximum;

}

class Snow {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Snow_ {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Speed {

    @SerializedName("Value")
    @Expose
    public Float value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Speed_ {

    @SerializedName("Value")
    @Expose
    public Float value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Speed__ {

    @SerializedName("Value")
    @Expose
    public Float value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Speed___ {

    @SerializedName("Value")
    @Expose
    public Float value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Sun {

    @SerializedName("Rise")
    @Expose
    public String rise;
    @SerializedName("EpochRise")
    @Expose
    public Integer epochRise;
    @SerializedName("Set")
    @Expose
    public String set;
    @SerializedName("EpochSet")
    @Expose
    public Integer epochSet;

}



class TotalLiquid {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class TotalLiquid_ {

    @SerializedName("Value")
    @Expose
    public Integer value;
    @SerializedName("Unit")
    @Expose
    public String unit;
    @SerializedName("UnitType")
    @Expose
    public Integer unitType;

}

class Wind {

    @SerializedName("Speed")
    @Expose
    public Speed speed;
    @SerializedName("Direction")
    @Expose
    public Direction direction;

}

class WindGust {

    @SerializedName("Speed")
    @Expose
    public Speed_ speed;
    @SerializedName("Direction")
    @Expose
    public Direction_ direction;

}

class WindGust_ {

    @SerializedName("Speed")
    @Expose
    public Speed___ speed;
    @SerializedName("Direction")
    @Expose
    public Direction___ direction;

}

class Wind_ {

    @SerializedName("Speed")
    @Expose
    public Speed__ speed;
    @SerializedName("Direction")
    @Expose
    public Direction__ direction;

}

File: app/src/main/java/com/example/testlunu/net/acu/weather/decompiled/WeatherWrapper.kt
package com.example.testlunu.net.acu.weather.decompiled

import com.example.testlunu.net.acu.weather.DailyForecast
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class WeatherWrapper (
    @SerializedName("Headline")
    @Expose
    var headline: Headline?,

    @SerializedName("DailyForecasts")
    @Expose
    var dailyForecasts: List<DailyForecast>?
)





internal class Direction {
    @SerializedName("Degrees")
    @Expose
    var degrees: Int? = null
    @SerializedName("Localized")
    @Expose
    var localized: String? = null
    @SerializedName("English")
    @Expose
    var english: String? = null
}

class Direction_ {
    @SerializedName("Degrees")
    @Expose
    var degrees: Int? = null
    @SerializedName("Localized")
    @Expose
    var localized: String? = null
    @SerializedName("English")
    @Expose
    var english: String? = null
}




class Headline {
    @SerializedName("EffectiveDate")
    @Expose
    var effectiveDate: String? = null
    @SerializedName("EffectiveEpochDate")
    @Expose
    var effectiveEpochDate: Int? = null
    @SerializedName("Severity")
    @Expose
    var severity: Int? = null
    @SerializedName("Text")
    @Expose
    var text: String? = null
    @SerializedName("Category")
    @Expose
    var category: String? = null
    @SerializedName("EndDate")
    @Expose
    var endDate: String? = null
    @SerializedName("EndEpochDate")
    @Expose
    var endEpochDate: Int? = null
    @SerializedName("MobileLink")
    @Expose
    var mobileLink: String? = null
    @SerializedName("Link")
    @Expose
    var link: String? = null
}

class Heating {
    @SerializedName("Value")
    @Expose
    var value: Int? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}

class Ice {
    @SerializedName("Value")
    @Expose
    var value: Int? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}

class Ice_ {
    @SerializedName("Value")
    @Expose
    var value: Int? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}



class Maximum__ {
    @SerializedName("Value")
    @Expose
    var value: Float? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}





class Moon {
    @SerializedName("Rise")
    @Expose
    var rise: String? = null
    @SerializedName("EpochRise")
    @Expose
    var epochRise: Int? = null
    @SerializedName("Set")
    @Expose
    var set: String? = null
    @SerializedName("EpochSet")
    @Expose
    var epochSet: Int? = null
    @SerializedName("Phase")
    @Expose
    var phase: String? = null
    @SerializedName("Age")
    @Expose
    var age: Int? = null
}

class Night {
    @SerializedName("Icon")
    @Expose
    var icon: Int? = null
    @SerializedName("IconPhrase")
    @Expose
    var iconPhrase: String? = null
    @SerializedName("HasPrecipitation")
    @Expose
    var hasPrecipitation: Boolean? = null
    @SerializedName("ShortPhrase")
    @Expose
    var shortPhrase: String? = null
    @SerializedName("LongPhrase")
    @Expose
    var longPhrase: String? = null
    @SerializedName("PrecipitationProbability")
    @Expose
    var precipitationProbability: Int? = null
    @SerializedName("ThunderstormProbability")
    @Expose
    var thunderstormProbability: Int? = null
    @SerializedName("RainProbability")
    @Expose
    var rainProbability: Int? = null
    @SerializedName("SnowProbability")
    @Expose
    var snowProbability: Int? = null
    @SerializedName("IceProbability")
    @Expose
    var iceProbability: Int? = null
    @SerializedName("Wind")
    @Expose
    var wind: Wind_? = null
    @SerializedName("WindGust")
    @Expose
    var windGust: WindGust_? = null
    @SerializedName("TotalLiquid")
    @Expose
    var totalLiquid: TotalLiquid_? = null
    @SerializedName("Rain")
    @Expose
    var rain: Rain_? = null
    @SerializedName("Snow")
    @Expose
    var snow: Snow_? = null
    @SerializedName("Ice")
    @Expose
    var ice: Ice_? = null
    @SerializedName("HoursOfPrecipitation")
    @Expose
    var hoursOfPrecipitation: Int? = null
    @SerializedName("HoursOfRain")
    @Expose
    var hoursOfRain: Int? = null
    @SerializedName("HoursOfSnow")
    @Expose
    var hoursOfSnow: Int? = null
    @SerializedName("HoursOfIce")
    @Expose
    var hoursOfIce: Int? = null
    @SerializedName("CloudCover")
    @Expose
    var cloudCover: Int? = null
}

class Rain {
    @SerializedName("Value")
    @Expose
    var value: Int? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}

class Rain_ {
    @SerializedName("Value")
    @Expose
    var value: Int? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}

class RealFeelTemperature {
    @SerializedName("Minimum")
    @Expose
    var minimum: Minimum_? = null
    @SerializedName("Maximum")
    @Expose
    var maximum: Maximum_? = null
}

class RealFeelTemperatureShade {
    @SerializedName("Minimum")
    @Expose
    var minimum: Minimum__? = null
    @SerializedName("Maximum")
    @Expose
    var maximum: Maximum__? = null
}

class Snow {
    @SerializedName("Value")
    @Expose
    var value: Int? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}

class Snow_ {
    @SerializedName("Value")
    @Expose
    var value: Int? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}




class TotalLiquid_ {
    @SerializedName("Value")
    @Expose
    var value: Int? = null
    @SerializedName("Unit")
    @Expose
    var unit: String? = null
    @SerializedName("UnitType")
    @Expose
    var unitType: Int? = null
}

class WindGust {
    @SerializedName("Speed")
    @Expose
    var speed: Speed_? = null
    @SerializedName("Direction")
    @Expose
    var direction: Direction_? = null
}

class WindGust_ {
    @SerializedName("Speed")
    @Expose
    var speed: Speed___? = null
    @SerializedName("Direction")
    @Expose
    var direction: Direction___? = null
}

class Wind_ {
    @SerializedName("Speed")
    @Expose
    var speed: Speed__? = null
    @SerializedName("Direction")
    @Expose
    var direction: Direction__? = null
}